crates/cli/src/main.rs

```diff
@@ -1,9 +1,6 @@
 use anyhow::Result;
 
-pub fn main() -> Result<()> {
-    tokio::runtime::Builder::new_current_thread()
-        .enable_all()
-        .build()
-        .unwrap()
-        .block_on(pacquet_cli::run_commands())
+#[tokio::main(flavor = "current_thread")]
```

Comment:
The message probably came from me. One off macros are fine, it is the super repetitive, less tuned macros that cause trouble.

Reply:
> macros increase the compilation time of Rust

This is true in the sense that they generate code, and the more code that exists, the more time it takes to compile.

> and should be avoided (?)

This is a bit strong. Every single `#[derive]` is a macro, for example. they are borderline impossible to truly avoid.

> If yes, is this change worth the increase? 

The amount of code generated by the macro is the code you'd write by hand. The increase you end up with here should be imperceptible.

> How can I benchmark/track?

One way to decide is to run some builds, then apply this diff, then run some builds. It's a very small diff so that shouldn't be difficult.